Mathlib/SetTheory/Ordinal/Nimber.lean

```diff
@@ -357,4 +361,32 @@ theorem add_trichotomy {a b c : Nimber} (h : a + b + c ≠ 0) :
   · rw [← hx'] at hx
     exact Or.inr <| Or.inr hx
 
+theorem add_nat (a b : ℕ) : ∗a + ∗b = ∗(a ^^^ b) := by
```

Comment:
Is it possible to deduce this from `grundyValue_nim_add_nim`, or maybe the other way around?

Reply:
We should be doing this the other way around. I plan to redefine `grundyValue` as a nimber in the near future, and to prove the more general result `grundyValue (nim a + nim b) = ∗a + ∗b`.

Reply:
I agree this is a good way round - do you want to add a TODO in this file saying to define grundyValue as a nimber?

Reply:
Great!